wingtips-zipkin/README.md

```diff
@@ -0,0 +1,88 @@
+* **`localComponentNamespace`** - The `zipkin.Constants.LOCAL_COMPONENT` namespace that will be used when creating certain Zipkin annotations when the Wingtips span's `Span.getSpanPurpose()` is `LOCAL_ONLY`. See the `zipkin.Constants.LOCAL_COMPONENT` javadocs for more information on what this is and how it's used by the Zipkin server, so you know what value you should send.
+* **`postZipkinSpansBaseUrl`** - The base URL of the Zipkin server. This should include the scheme, host, and port (if non-standard for the scheme). e.g. `http://localhost:9411`, or `https://zipkinserver.doesnotexist.com/`.
+
+## Using Native Zipkin Brave `SpanCollector`s
+
+The default `ZipkinSpanSender` implementation provided by this submodule (used under the hood when you follow the Quickstart instructions above) for sending spans to the Zipkin server should work decently for many use cases, however it is limited to HTTP transport and does not have some of the features provided by Zipkin `SpanCollector`s, which serve a similar purpose.
+
+Zipkin's [Brave](https://github.com/openzipkin/brave) libraries provide numerous Zipkin `SpanCollector` implementations that you could use as a Wingtips `ZipkinSpanSender` by creating a simple adapter that implements the `ZipkinSpanSender` interface but uses the concrete Zipkin Brave `SpanCollector` under the hood to do the work.
+
+Simply pull in the Zipkin Brave dependencies into your project that are necessary to have access to the `SpanCollector` implementation you want to use, and create an adapter like the following:
+
+```java
+public class BraveSpanCollectorAdapter implements ZipkinSpanSender {
+
+    private final Logger logger = LoggerFactory.getLogger(this.getClass());
+    private final HttpSpanCollector braveCollector;
+    private final ExecutorService forcePurgeExecutor = Executors.newSingleThreadExecutor();
+
+    public BraveSpanCollectorAdapter(HttpSpanCollector braveCollector) {
+        this.braveCollector = braveCollector;
+    }
+
+    @Override
+    public void handleSpan(zipkin.Span span) {
+        try {
+            braveCollector.collect(toBraveSpan(span));
+        } catch (Throwable ex) {
+            logger.warn("An error occurred while adding a span to the Brave collector.", ex);
+        }
+    }
+
+    @Override
+    public void forcePurgeBatchedSpans() {
+        forcePurgeExecutor.execute(new Runnable() {
+            @Override
+            public void run() {
+                try {
+                    braveCollector.flush();
+                } catch (Throwable ex) {
+                    logger.warn("An error occurred while flushing the Brave collector.", ex);
+                }
+            }
+        });
+    }
+
+    public com.twitter.zipkin.gen.Span toBraveSpan(zipkin.Span zipkinSpan) {
+        throw new UnsupportedOperationException("Implementation of this method left as an exercise to the reader");
```

Comment:
@adriancole I saw brave span -> zipkin span code in the brave libraries, but nothing for the reverse, so I punted. =) If there is something for this that I missed or just a better way to do it then please point me at the solution. Seems a little awkward to do all this wingtips->canonical zipkin->brave span model conversion but it ultimately works fine. Would be nice if the brave `SpanCollector`s accepted the canonical zipkin model as well as the brave model, but probably not worth a bunch of refactoring just for this use case. Let me know what you think.